integration/docker/docker_s390x.go

```diff
@@ -0,0 +1,8 @@
+//+build s390x
```

Comment:
Nit: not required as you've already encoded the platform in the name of the file itself.